includes/User.php

```diff
@@ -2566,6 +2629,8 @@ public function getLocalFlag($flag, $cityId = null, $sep = "-") {
 		if (!isset($cityId)) {
 			$cityId = $wgCityId;
 		}
```

Comment:
Is this block still necessary since it's handled [below](https://github.com/Wikia/app/pull/7723/files#diff-897fba1dd6b8edc3238762e35d0b7c71R2668)?